exportMarkdown.js

```diff
@@ -12,7 +12,7 @@ exports.getPadMarkdown = (pad, revNum, callback) => {
 
     // fetch revision atext
     (callback) => {
-      if (revNum !== undefined) {
+      if (revNum !== null) {
```

Comment:
Good find. I suggest using loose comparison to `null` because we want `undefined` to be treated the same as `null`, and it's more idiomatic:

```suggestion
      if (revNum != null) {
```